packer/linux_debian.pkr.hcl

```diff
@@ -1,34 +1,28 @@
 variable "image_date" { type = string }
 variable "gcp_project" { type = string }
-variable "task_name" { type = string }
-
-variable "prefix" {
-  type = string
-  default = ""
-}
+variable "image_name" { type = string }
 
 locals {
-  name = "${var.prefix}pg-ci"
-  image_identity = "${local.name}-${var.task_name}-${var.image_date}"
+  image_identity = "${var.image_name}-${var.image_date}"
 
   debian_gcp_images = [
     {
-      name = "bullseye"
+      source_name = "bullseye"
       zone = "us-west1-a"
       machine = "e2-highcpu-4"
     },
     {
-      name = "sid"
+      source_name = "sid"
       zone = "us-west1-a"
       machine = "e2-highcpu-4"
     },
     {
-      name = "sid-newkernel"
+      source_name = "sid-newkernel"
       zone = "us-west2-a"
       machine = "c2-standard-8"
     },
     {
-      name = "sid-newkernel-uring"
+      source_name = "sid-newkernel-uring"
```

Comment:
What does "source_name" really mean? With "source" I'd imagine it to reference the original image or such, but clearly this isn't that.

Reply:
It is packer's build's source_name; I found the 'name' itself confusing so I changed it to source_name. Does 'build_source_name/id' or such work?

Reply:
I don't know what "packer's build's source_name" means?

Reply:
[This source part](https://github.com/anarazel/pg-vm-images/blob/35e3a3e9fd1ffd225059d820c63924aaa2671c72/packer/linux_debian.pkr.hcl#LL53C1-L65C4)

I mean packer files have source part and we set this source part's name using dictionaries(`debian_gcp_images` on this example). We set these sources to match with `packer build -only "*.${TASK_NAME}" `.

I think it being only stated as `name` is confusing since we also have `IMAGE_NAME` variable and this `name` is not same as `IMAGE_NAME`. So, I wanted to change it  to `source_name` since it is only used while we dynamically set source part.

Reply:
I think they need to be same with `TASK_NAME`, maybe we can set these variables as `task_name` instead of `source_name`?